### PR TITLE
More respect readable and semantics code.

### PR DESCRIPTION
**Pull request by hrsh7th**

### CODE REVIEW

File: src/component/RoadmapCanvas.js [hunk 1]
```diff
@@ -227,54 +227,44 @@ export class RoadmapCanvas extends EventEmitter {
...
+      .selectAll('svg')
       .data(tasks)
       .enter()
-      .append('g')
-      .attr('class', 'bar');
-    bar.append('rect')
+      .append('svg');
+
+    // create bar boxes.
+    bars
```
Comment (hrsh7th) [inline src/component/RoadmapCanvas.js:239]: It's svg element array.
svg has relative positioning context.

File: src/component/RoadmapCanvas.js [hunk 2]
```diff
@@ -227,54 +227,44 @@ export class RoadmapCanvas extends EventEmitter {
    * @private
    */
   _appendTaskLines(tasks, svg, xScale, yScale) {
```
Comment (atago0129) [inline src/component/RoadmapCanvas.js:229]: I want to rename to `_appendTaskBars`

File: src/component/RoadmapCanvas.js (hunk 1 above)
Comment (atago0129) [reply src/component/RoadmapCanvas.js:239]: cool

**Comment by atago0129** [comment]
duplicate https://github.com/atago0129/roadmappy/pull/19

**Pull request closed without merging**